pipeline-modules/deployment-service/gcp/basic-gke-regional.yaml

```diff
@@ -284,5 +289,10 @@ template: |
     - |
       cd {{helm_chart_path}} ;
       helm list;
-      helm uninstall {{service_name}};
+      helm uninstall {{service_name}};      
   {% endif %}
+  {% if cloudbuild_private_worker_pool != "" %}    
+  options:
```

Comment:
The options are defined in common pipeline-config module https://github.com/cldcvr/codepipes-ci-cd-modules/blob/60ab0c3020aeda67f3994e4a2476f66b85beb09b/pipeline-modules/common/gcp/pipeline-config.yaml#L140

Things like disk-size are already there. Since these are "common" settings that apply to the entire pipeline individual modules cannot override them as the modules would end up stepping on each other.

You will need to add the workpool definition and input to the common file instead.

The individual modules can only add values not already defined (e.g. they can add more ENV variables to the env map) but two modules must not conflict on value.

Reply:
Just call it `worker_pool` instead of `cloudbuild_private_worker_pool` as that is what it really is and can be set for any template.